tutorials/simulators/OpenFOAM.md

```diff
@@ -105,6 +105,42 @@ and place them in a folder named `highLiftConfiguration`.
    24 -rw-rw-r--@  1   11399 Jun 13 10:09 thumbnail.png
    ```
 
+### Step 0: An overview
```

Comment:
@Pbarbosa92 just add the title: Overview

Reply:
Done